File: Python - Divers/GinFizz.py
def gin_fizz(numberIterations):
    if numberIterations==0:
        return

    if numberIterations % 15==0 :
        print('{} - Gin-Fizz-Buzz'.format(numberIterations))
    else:
        if numberIterations % 5 ==0:
            print('{} - Fizz'.format(numberIterations))
        else:
            if numberIterations % 3 ==0:
                print('{} - Buzz'.format(numberIterations))
            else:
                print('{} - Gin'.format(numberIterations))
    gin_fizz(numberIterations-1)

gin_fizz(100)    

File: C# - Visual Studio/GinFizz.cs
int compteur;

for (compteur=0,compteur<100,compteur++)
    {
        if (compteur%15 == 0)
            {Console.WriteLine("GinFizzBuzz")}
        else { 
            if(compteur%5 == 0)
                {Console.WriteLine("Fizz")}
            else{
                if(compteur%3 == 0)
                        {Console.WriteLine("Buzz")}
                else{
                        Console.WriteLine("Gin")}
                }
                }
    }
    

    